resolwe_bio/docker_images/rnaseq/packages-manual/trimmomatic.sh

```diff
@@ -10,12 +10,19 @@ download_and_verify \
     http://www.usadellab.org/cms/uploads/supplementary/Trimmomatic/Trimmomatic-\${version}.zip \
     Trimmomatic-\${version}
 
-cat >trimmomatic.sh <<EOL
+cat >trimmomaticSE <<EOL
 #!/bin/bash
 
-java -jar /opt/usadellab/trimmomatic/trimmomatic-0.36.jar "$@"
+java -jar /opt/usadellab/trimmomatic/trimmomatic-0.36.jar SE "\$@"
```

Comment:
You could avoid this and use `'EOF'` to disable interpretation of variables in the whole _here_ document. See [this StackOverflow answer](https://stackoverflow.com/a/2954835/1872673) for more details.